### ISSUE TITLE
F10

### ISSUE THREAD

**Issue by Conte-harebourd** [Azzary/DofusMods#1]
F10 key does not work, it does not open the control panel to activate plugins

**Comment by Azzary** [comment]
The panel is open by default. The installation may have failed. Do you have a console open in addition to Dofus?

**Comment by Conte-harebourd** [comment]
yes
![Mods dofus](https://github.com/user-attachments/assets/a5ec63c4-4fd7-4585-8037-6bc78131eb7a)

**Comment by Conte-harebourd** [comment]
![thumbnail_image](https://github.com/user-attachments/assets/d0fa0b2a-7049-4698-9f7c-f93d281049f1)

**Comment by Azzary** [comment]
Yes, normally, it shouldn't cause any problems. I have the same errors, and there is indeed:
[Info : Dofus Mods Plugin] The Dofus Mods Plugin is loaded!
Maybe the window is appearing off-screen. I have a very large screen. I will try with another screen

**Comment by Conte-harebourd** [comment]
Je suis sur un pc portable de 15 pouce
